### PR TITLE
 Create an instant App version of PDF Converter #496

### PR DESCRIPTION
**Pull request by Vanethos**
Create an instant App version of PDF Converter #496

Added Base, Install and Instant Modules
Added support for Butterknife 9 to let butterknife run from a module
Refactored code to accomodate module change
Refactored build.gradle files so that we declare the dependencies version in the project's build.gradle file

# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

Fixes #(issue)

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [ ] `./gradlew checkstyle`

# Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

**Comment by Swati4star** [comment]
@Vanethos There are a few conflicts. Please look into those.

**Comment by Vanethos** [comment]
Fixing build issue now

**Comment by Vanethos** [comment]
On my end, the app successfully builds, though I'm seeing that Travis CI build failed 🤔 

```
Failed to install the following Android SDK packages as some licences have not been accepted.
     build-tools;28.0.3 Android SDK Build-Tools 28.0.3
     platforms;android-28 Android SDK Platform 28
  To build this project, accept the SDK license agreements and install the missing components using the Android Studio SDK Manager.
  Alternatively, to transfer the license agreements from one workstation to another, see http://d.android.com/r/studio-ui/export-licenses.html
```

File: base/build.gradle [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
 apply from: '../config/checkstyle.gradle'
+/**
+ * In order to use Butterknife in a Library module, we must do the following:
+ * https://github.com/JakeWharton/butterknife#library-projects
+ */
+apply plugin: 'com.jakewharton.butterknife'
+
 
 android {
-    compileSdkVersion 27
```
Comment (Swati4star) [inline base/build.gradle]: you have changed compile version from 27 to 28. So, you got to update the build version here also: 
https://github.com/Swati4star/Images-to-PDF/blob/master/.travis.yml#L7

Comment (Vanethos) [reply base/build.gradle]: I'm sorry, I created a new PR without knowing it!

https://github.com/Swati4star/Images-to-PDF/pull/501

Changed build tools and compile sdk

**Comment by Swati4star** [comment]
Why did you create a different PR https://github.com/Swati4star/Images-to-PDF/pull/501 for the fix?

File: base/build.gradle (hunk 1 above)
Comment (Swati4star) [reply base/build.gradle]: You should make those changes a part of this PR

**Comment by Swati4star** [comment]
Closing this one because of inactivity. Feel free to reopen it, if you wish to work on this one 👍

**Pull request closed without merging**